.devcontainer/local/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "nextflow-training (local)",
+
+    // Update this to pin the release tag before release
+    "image": "ghcr.io/nextflow-io/training:latest",
+    // Uncomment this and comment "image" when testing Dockerfile changes
+    // "build": { "dockerfile": "Dockerfile", "context": ".." },
+
+    "features": {
+        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {
+            "version": "latest"
+        }
+    },
+    "workspaceFolder": "${localWorkspaceFolder}",
+    "workspaceMount": "source=${localWorkspaceFolder},target=${localWorkspaceFolder},type=bind",
+    "remoteEnv": {
+        "NXF_HOME": "/workspaces/training/.nextflow",
+        "HOST_PROJECT_PATH": "/workspaces/training"
+    },
+    "postCreateCommand": "sudo mkdir -p /workspaces && sudo ln -s ${localWorkspaceFolder} /workspaces/training && if [ -z \"$CODESPACES\" ]; then echo \"Local Development\"; else echo \"Codespaces Development\"; fi",
```

Comment:
Could we get rid of the `if [ -z \"$CODESPACES\" ]; then echo \"Local Development\"; else echo \"Codespaces Development\"; fi` bit?